src/Microsoft.AspNetCore.Mvc.Razor.Host/InjectDirective.cs

```diff
@@ -20,13 +20,13 @@ public static IRazorEngineBuilder Register(IRazorEngineBuilder builder)
             return builder;
         }
 
-        internal class Pass : IRazorIRPass
+        internal class Pass : IRazorDirectiveClassifierPass
         {
             public RazorEngine Engine { get; set; }
 
-            public int Order => RazorIRPass.DirectiveClassifierOrder;
+            public int Order => 50;
```

Comment:
Why isn't it 0?

**HINT**

I'm going to ask this question every time I see a non-default value. Do yourself a favor and always make a comment in the code before sending a PR.